solutions/devsprint-michel-ferreira-2/src/main/kotlin/io/devpass/parky/service/ParkingSpotService.kt

```diff
@@ -17,4 +17,8 @@ class ParkingSpotService(
     fun findAllParkingSpot(): List<ParkingSpot> {
```

Comment:
Como a lógica ficará aqui, o seu método deve receber o parâmetro "inUse" (que pode ser nulo) passado pelo Controller, e então chamar o método do repositório corretamente, sem fazer o filtro: "filter { it.inUseBy == null }"

Reply:
Haverá três situações:
"inUse"  == null -> Buscar todas as vagas;
"inUse"  == true -> Buscar todas as vagas em uso (criar um novo método no ParkingSpotRepository ;
"inUse"  == false -> Buscar todas as vagas disponíveis (criar o método sugerido acima);